includes/admin/load.php

```diff
@@ -375,3 +377,33 @@ function addPluginProgressIndicator( message ) {
 		array( 'type' => 'module' )
 	);
 }
+
+/**
+ * Callback function that returns plugin settings link.
+ *
+ * @param string $plugin_slug Plugin slug passed to generate the settings link.
+ * @return string Either the plugin settings link or empty.
+ * @since n.e.x.t
```

Comment:
```suggestion
 * @since n.e.x.t
 *
 * @param string $plugin_slug Plugin slug passed to generate the settings link.
 * @return string Either the plugin settings link or empty.
```